app/templates/header.html

```diff
@@ -23,7 +23,7 @@
             </a>
         </div>
         <div class="govuk-header__content">
-            <span class="app-service-name govuk-header__link--service-name">{{ service_title }}</span>
+            <span class="app-service-name govuk-header__link--service-name">{{ gettext("Access Funding") if is_error else service_title or get_service_title() }}</span>
```

Comment:
What's going on here in this cascade? When is `service_title` defined rather than calling `get_service_title()`? Does it get cached?

Reply:
Not the prettiest situation.

`@application_bp.route("/tasklist/<application_id>", methods=["GET"])` provides a `service_title` to the template.
(no idea why, I just maintained the existing logic)

Before, it used to override the `service_title` from the `context_processor`, as they had the same variable name. And it worked fine, because both were strings.

Now we cannot do that anymore because the `utility_processor` provides a function that we need to invoke in the template.